hgtv/uploads.py

```diff
@@ -25,17 +25,16 @@ def return_werkzeug_filestorage(request, filename):
     if extension not in current_app.config['ALLOWED_EXTENSIONS']:
         raise UploadNotAllowed("Unsupported file format")
     new_filename = secure_filename(filename + '.' + extension)
-    try:
-        tempfile = StringIO(buf=request.content)
-    except AttributeError:
-        if hasattr(request.stream, 'getvalue'):
-            tempfile = StringIO(buf=request.stream.getvalue())
-        elif hasattr(request.stream, 'read'):
-            tempfile = StringIO(buf=request.stream.read())
+    if hasattr(request, 'content'):
+        tempfile = BytesIO(request.content)
+    elif hasattr(request, 'read'):
```

Comment:
Should be `else`, or you could end up with no `tempfile`.

Reply:
I suspect this entire block could be better. We shouldn't be doing guesswork here.